src/util/response.js

```diff
@@ -0,0 +1,21 @@
+export function createResponseMessage(type, success, data, token){
+    let message = "Invalid Data";
+    if (success == true){
+        switch(type){
+            case 'post':
+                message = "Data created successfully";
+                break;
+            case 'get':
+                message = 'Data found';
```

Comment:
might as well just append "successfully" to the end of this as well

Reply:
yea to make it consistent with all the other messages why not lol